lib/oper.g

```diff
@@ -1758,8 +1782,24 @@ BIND_GLOBAL( "InstallGlobalFunction", function( arg )
         func := arg[2];
     fi;
     if IS_STRING( oper ) then
-      oper:= VALUE_GLOBAL( oper );
+      oper := VALUE_GLOBAL( oper );
     fi;
+
+ok := true;                                             ## for Utils 
+if ( oper in UTILS_FUNCTION_OPERS ) then                ## for Utils 
+    pos := 1;                                           ## for Utils 
+    while not ( oper = UTILS_FUNCTION_OPERS[pos] ) do   ## for Utils 
+        pos := pos + 1;                                 ## for Utils 
+    od;                                                 ## for Utils 
+    val := UTILS_FUNCTION_COUNT[pos];                   ## for Utils 
+    if ( val >= 2 ) then                                ## for Utils 
+        ok := false;                                    ## for Utils 
+    fi;                                                 ## for Utils 
+##  Print( "#Ui: ", UTILS_FUNCTION_NAMES[pos],          ## for Utils  
+##         ", ", val, ", ok = ", ok, "\n" );            ## for Utils 
+fi;                                                     ## for Utils 
+
+if ok then                                              ## for Utils 
```

Comment:
For this bit, I assume the lack of indention is meant to avoid re-indenting the code wrapped inside the if/endif. But couldn't you achieve the same effect by writing
```
if not ok then return; fi;
```
That could then be indented correctly.